content/blog/2020-12-24-darktable-3.4/2020-12-24-darktable-3.4.md

```diff
@@ -0,0 +1,182 @@
+title: darktable 3.4: Encore!
+author: the darktable team
+slug: darktable-3-4
+date: 2020-12-24
+lede: --
+lede_author: --
+tags: announcement, darktable-release
+
+Happy holidays everyone -- it's time for your favourite Christmast gift. This is the second major release of 2020 from the darktable project following the early release of darktable 3.2 in August, and we've been busy.
+
+# Documentation
+
+Photography can be obstuse and trying to manage your post-processing without documentation can make things even harder!  This time, though, the darktable team has been busy getting the user manual ready in time for the release and it is available today at https://www.darktable.org/resources/, and fully up-to-date with the latest version.
```

Comment:
Do you mean "abstruse" here?

Reply:
I was going for "difficult to comprehend" which obtuse means, but it isn't the first definition.

Reply:
I don't think that's the right definition. You can call a person obtuse meaning that they are slow.

Reply:
Changed.